### PR TITLE
chore: update hydra from v1.8.5 to v1.10.5

### PR DESCRIPTION
**Pull request by pommelinho**
## Proposed changes

Update hydra from v1.8.5 to v1.10.5

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

**Comment by Demonsthere** [comment]
Hello there!
Please also update the version in the [Chart.yaml](https://github.com/ory/k8s/blob/master/helm/charts/hydra/Chart.yaml#L2)
and sync the changes in the [CI values](https://github.com/ory/k8s/blob/master/.circleci/values/hydra.yaml#L3).
I think we can delete or comment the overrides now :)

File: .circleci/values/hydra.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 image:
   repository: oryd/hydra
-  tag: v1.10.3-alpine
+  tag: v1.10.5-alpine
```
Comment (Demonsthere) [inline .circleci/values/hydra.yaml:3]:
```suggestion
  # tag: v1.10.5-alpine
```

Comment (Demonsthere) [reply .circleci/values/hydra.yaml:3]: Please comment or delete this part (the whole `image` section), if we update the default version in the chart, then this overrides makes no sense anymore :)

Comment (pommelinho) [reply .circleci/values/hydra.yaml:3]: yes you are right ;-) changed it!!

**Review by Demonsthere — APPROVED** [review]
👍

**Pull request merged**